server/src/server.rs

```diff
@@ -881,7 +903,12 @@ impl Server {
         tokio::task::spawn_local(async move { server_forward_resumer.forward_resumer().await });
 
         while let Some((packet, addr, timestamp)) = input.next().await {
-            counter!("ya-relay.packets.incoming", 1);
+            counter!("ya-relay.packet.incoming", 1);
+
+            if server.drop_policy(&packet, timestamp) {
+                counter!("ya-relay.packet.dropped", 1);
+                continue;
```

Comment:
Can we have logs for this? Querying metrics via an external service is usually not the best experience to say at least